docs/pages/application-access/guides/dynamodb.mdx

```diff
@@ -1,22 +1,24 @@
 ---
-title: Database Access with AWS DynamoDB
-description: How to access AWS DynamoDB with Teleport.
+title: Access AWS DynamoDB as an Application
+description: How to access AWS DynamoDB as an Application with Teleport.
 ---
 
-Access to AWS DynamoDB is provided by [**Teleport Application
+Access to AWS DynamoDB can be provided by [**Teleport Application
 Access**](../../application-access/introduction.mdx) for the AWS Console and
-API.
+API. This is an alternative to accessing DynamoDB through the Teleport Database
+service, as described in our [Database Access with AWS DynamoDB](../../database-access/guides/aws-dynamodb.mdx) guide.
```

Comment:
Before I read any further, I probably want to understand if I'm on the right guide or if I should click this link and go to the DB access guide.

Maybe a small section describing the pros and cons of each approach (on both pages) would be helpful here?

Reply:
I agree with you, but lack the subject matter expertise to provide the requested details. Maybe @smallinsky can suggest some points that I can add here?

Reply:
The DynamoDB thought database access mostly works the same but it provides the unified experience with other databases protocols like distinguishable audit logs  and ability to connect through `tsh proxy db`.

Reply:
Main benefit is db access can be used by GUI clients like NoSQL workbench (not possible via app access)

Reply:
See 0fdbfce9c6a73ad77d11b7701b36d4c7ffc73f7d